plugins/ngrok/provisioner.go

```diff
@@ -95,6 +116,31 @@ func getConfigValueAndNewArgs(args []string, newFilePath string) (string, []stri
 	return "", append(args, fmt.Sprintf("--config=%s", newFilePath))
 }
 
+// Get installed ngrok version and required version
+func getNgrokVersion() (string, string, error) {
+	cmd := exec.Command("ngrok", "--version")
+	ngrokVersion, err := cmd.Output()
+	if err != nil {
+		return "", "", errors.New("ngrok not found")
+	}
+
+	// Example: "ngrok version 3.1.1\n" to "3.1.1\n"
+	currentVersion := strings.TrimPrefix(string(ngrokVersion), "ngrok version ")
+
+	// Example: "3.1.1\n" to "3.1.1"
+	currentVersion = strings.Trim(currentVersion, "\n")
+
+	// Example: "3.1.1" to "v3.1.1" as that's the format
+	// the package semver expects
+	currentVersion = "v" + currentVersion
+
+	// NGROK_API_KEY is supported only from ngrok 3.2.1
+	// NGROK_AUTH_TOKEN was already supported, but NGROK_API_KEY is a new addition
+	requiredVersion := "v3.2.1"
```

Comment:
Given that this is a constant version that won't change, I think it makes more sense to make it a `const` at the top of the file and call it something like `envVarAuthVersion` to make it clear that we are using it to determine when we can use env var provisioning.